### PR TITLE
docs: add account id info

### PR DESCRIPTION
**Pull request by ianaya89**
## Affected Components
* [ ] Resources
* [ ] Tests
* [x] Docs
* [ ] Other

## Style
* [ ] Terraform code is formatted with `terraform fmt`
* [ ] Go code is formatted with `go fmt`

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
+$ export TF_VAR_checkly_account_id="your-account-id"
+```
 
 ## Example usage
 
 ```terraform
-# define a variable to hold the API Key
+# define a variable to hold the API Key and Account ID
 variable "checkly_api_key" {}
+variable "checkly_account_id" {}
```
Comment (ragog) [inline docs/index.md:23]: the provider version below should say '1.3.0', right? @ianaya89

**Review by ragog — CHANGES_REQUESTED** [review]

File: docs/guides/getting-started.md [hunk 2]
```diff
@@ -55,4 +56,4 @@ resource "checkly_check_group" "first-group" {
 }
 ```
 
-5. Now run `$ terraform init` to setup a new project. Then use `$ terraform plan` followed by `$ terraform apply` to deploy your monitoring.
+6. Now run `$ terraform init` to setup a new project. Then use `$ terraform plan` followed by `$ terraform apply` to deploy your monitoring.
```
Comment (ragog) [inline docs/guides/getting-started.md:59]: @ianaya89 also this should say just 5, right?

Comment (ianaya89) [reply docs/guides/getting-started.md:59]: Mmm not sure if I get you, we have 6 steps.

Comment (ianaya89) [reply docs/guides/getting-started.md:59]: Mmm not sure if I get you, we have 6 steps.

Comment (ragog) [reply docs/guides/getting-started.md:59]: @ianaya89 forget about it, I didn't see the ellipsis in the diff. All good!

**Review by ragog — APPROVED** [review]
LGTM

**Pull request merged**